README.md

```diff
@@ -128,3 +128,23 @@ catch (e) {
 
 return ex;
 ```
+
+## Sentry error notification
+
+```js
+let ex = {};
+
+try {
+  simp.start();
```

Comment:
Just a couple notes, FYI (since there is zero documentation I understand you have no way of knowing this, so this is more a "by the way, did you know?". Except I know you didn't. :grin: )

You don't really need `simp.start()` here.

Every time you do `simp.find()` it automatically starts from the start of the document. The trick comes in that chained other calls actually keep your current position within the document (that's why `find()` and then e.g. `content()` works as expected, since the "pointer" is still pointing to the same item).

If you want to avoid this (`find()` restarting from the start of the document), you *could* store the reference object returned by `find()` and re-use it, but easier just to use `simp.continue()` which tells the library not to re-initialize the pointer pointing to the start of the document.

So you don't really need `start()` and `find()` initializes the search anyway, so why would you ever need `start()` anywhere? Well, it's a (perhaps inelegant) workaround for [this issue](https://github.com/flaki/simplayground#nintendo-eshop-items-on-your-wishlist-are-on-sale-notification) when you want to initialize your search once, just simply pointing to the first element of the document and then just use `continue()` in a loop to iterate through entries. Here, you can't just use `find()` alone since that would restart the search at the beginning of the document at each turn of the loop! You also can't just use `continue()` since there's nothing to *continue* :) This last thing, could perhaps be fixed by sprinking on some :sparkles: magic :sparkles:, but I think mandating `start()` before you intend to `continue()` is probably a good practice so that's why it's like that.

Again, start is not "harmful" in any way so this is not really an issue by any means. :)

Reply:
This is some stellar feedback, thank you! That explains some minor struggles I was having :)

I'll play around with the example and see if I can make it cleaner, then 😄 Thanks again!